### PR TITLE
boards: nordic/ezurio/vcc-gnd/ruuvi: Remove setting PINCTRL in defconfig files

### PR DESCRIPTION
**Pull request by nordicjm**
boards: nordic: Remove setting PINCTRL in defconfig files
    
    Removes setting this Kconfig in the wrong place

    boards: ezurio: rm1xx_dvk: Remove setting PINCTRL
    
    Removes setting this Kconfig in the wrong place

    boards: vcc-gnd: yd_stm32h750vb: Remove PINCTRL
    
    Removes setting this Kconfig in the wrong place

    boards: ruuvi: ruuvitag: Remove setting PINCTRL
    
    Removes setting this Kconfig in the wrong place

### CODE REVIEW

**Review by rerickson1 — DISMISSED** [review]

**Review by anangl — DISMISSED** [review]

**Review by henrikbrixandersen — DISMISSED** [review]

**Review by rerickson1 — DISMISSED** [review]

File: tests/drivers/build_all/eeprom/boards/nrf9131ek_nrf9131.conf
Comment (henrikbrixandersen) [inline tests/drivers/build_all/eeprom/boards/nrf9131ek_nrf9131.conf:1]: Should we split this bug fix into its own PR to speed up the process of getting it in?

Comment (nordicjm) [reply tests/drivers/build_all/eeprom/boards/nrf9131ek_nrf9131.conf:1]: Have removed the changes to the nrf9131 boards and will submit them separately later. Now, hopefully, the CI should pass...

**Review by henrikbrixandersen — APPROVED** [review]

**Review by anangl — APPROVED** [review]

**Review by rerickson1 — APPROVED** [review]
